pom.xml

```diff
@@ -244,9 +244,24 @@
         <artifactId>rfc5545-datetime</artifactId>
         <version>0.2.2</version>
       </dependency>
+
+      <dependency>
+        <groupId>com.amazonaws</groupId>
+        <artifactId>athena-jdbc</artifactId>
+        <version>1.0.0</version>
+        <scope>system</scope>
+        <systemPath>${project.basedir}/lib/AthenaJDBC41-1.0.0.jar</systemPath>
```

Comment:
I know this is gross, will figure out a better solution for this before it gets merged anywhere